solana-programs/Dockerfile

```diff
@@ -9,7 +9,7 @@ RUN apt-get update && \
     apt-get install -y jq curl build-essential libudev-dev libhidapi-dev pkg-config libssl-dev git python-is-python3 python3-pip && \
     pip3 install --no-cache-dir web3 && \
     curl -s --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y && \
-    sh -c "$(curl -sSfL https://release.solana.com/v1.7.8/install)"
```

Comment:
any reason we're doing different versions in CI vs solana Dockerfile?

Reply:
just trying to figure out which one works before modifying both